src/tools/rbd/action/Qos.cc

```diff
@@ -0,0 +1,120 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Yes, agree with store the key/value in image-metadata way. And I am doing that in this PR. But I did a wrap of the metadata_list and metadata_set for qos case.

https://github.com/ceph/ceph/pull/16865/files#diff-fec195624ee36961c2f7215f08529e28R1265 

In addition, the qos CLI is introduced to change the qos setting online. Actually, I want to say, we need a ```rbd resize``` to change the size of image. then do we need a ``` rbd qos ``` to change the qos setting and notify all wathers?